src/Application.nts

```diff
@@ -57,6 +58,29 @@ class Application extends Nullstack {
           <div class="max-w-6xl mx-auto grid gap-12">
             <Panel id="elements" title="Elements">
               <div class="grid gap-8">
+                <div>
+                  <h3 class="text-lg font-semibold text-gray-600 mb-2">Box</h3>
+                  <p class="mb-3 text-gray-400 text-sm">
+                    Add inline styles to any element or component with sx
+                  </p>
+                  <Preview class="bg-gray-100 m-2">
+                    <Box
+                      shadow="md"
+                      radius="md"
+                      withBorder
```

Comment:
@brunopoeta maybe we can remove the `border` property and simply pass the border styles via `class`. otherwise, we'll end up by needing to map all possible styles in this case.